horreum-backend/src/main/resources/changeDetection/hunter.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
+  horreum:
+    type: csv
+    file: tests/resources/horreum.csv
```

Comment:
Is it intended to keep test resource here?

Reply:
This isn't a test resource, it is a config that is required at runtime to configure hunter

Reply:
gotcha!